tests/test_virtual_chassis.py

```diff
@@ -24,3 +27,184 @@ def test_connectivity(self, vct):
                 _, out = dv.runcmd(['sh', "-c", "ping -c 5 -W 0 -q %s" % ip])
                 print(out)
                 assert '5 received' in out
+
+    def test_voq_switch(self, vct):
+        """ Test VOQ switch objects configuration """
```

Comment:
Two things:

1. I would remove the extra spaces around the comment, and
2. Could you give a brief summary of what object configuration we're checking in this method?

So, the doc comment should end up looking like this:
```suggestion
        """Test VOQ switch objects configuration.
        
        Insert brief summary here.
        """
```

Reply:
Fixed as suggested.